src/utils/init/utils.js

```diff
@@ -154,25 +106,24 @@ const getBuildSettings = async ({ config, env, repositoryRoot, siteRoot }) => {
       frameworkBuildDir,
       frameworkPlugins,
     })
-  const { baseDir, buildCmd, buildDir, functionsDir, installSinglePlugin, plugins } = await inquirer.prompt(
-    await getPromptInputs({
+
+  if (recommendedPlugins.length !== 0) {
+    log(`Configuring ${formatTitle(frameworkName)} runtime...`)
```

Comment:
```suggestion
    log(`Configuring ${formatTitle(frameworkName)}...`)
```

It has "runtime" in the plugin name already

Reply:
Oh wait, you're using the framework name not plugin name. Carry on!

Reply:
Yeah. I could use the plugin name, but theoretically you could have many plugins (even though I suspect in reality we never do). I'll use the framework for now, and we can revisit later.